pyscitt/pyscitt/client.py

```diff
@@ -577,6 +577,21 @@ def submit_claim_and_confirm(
         receipt = self.get_receipt(tx, decode=False)
         return Submission(operation_id, tx, receipt, False)
 
+    def submit_and_confirm(
```

Comment:
This is a duplicate of `submit_claim_and_confirm` above, why? Surely we can have just one method.

Reply:
Yes, I am in the process of fixing the naming in the test infra and consolidating. It's going to take a while, but do feel free to help.